src/webpack/shared.js

```diff
@@ -7,13 +7,13 @@ module.exports = (babelrc) => ({
       options: babelrc || {
         cacheDirectory: true,
         presets: [
-          'es2015',
+          'env',
           'react'
         ],
         plugins: [
           'transform-object-rest-spread',
           'syntax-dynamic-import',
-          'idx'
+          'babel-plugin-idx'
```

Comment:
I presume these work with or without babel-plugin as a prefix. Lets choose yay or nay.

Reply:
Oh yeah, my mistake, had some issues where `idx` wasn't getting stripped out from the bundle so followed the `idx` docs and they had the full reference. I'll update.